queries.sql

```diff
@@ -18,6 +18,41 @@ SELECT * from animals WHERE name <> 'Gabumon';
 
 SELECT * from animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
 
+Begin;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+Begin;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+commit;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=species%20already%20set.-,Verify%20that%20changes%20were%20made.,Verify%20that%20changes%20persist%20after%20commit.,-Take%20a%20screenshot), Please verify that changes were made and verify that changes persist after commit.

Reply:
- [ ] Again here, after setting the species column with values, kindly verify that the changes were made by using a `select` statement.
e.g
`SELECT species from animals;`